clang/lib/AST/Interp/Context.cpp

```diff
@@ -102,7 +102,8 @@ std::optional<PrimType> Context::classify(QualType T) const {
     case 8:
       return PT_Sint8;
     default:
-      return std::nullopt;
+      return PT_IntAPS;
+      // return std::nullopt;
```

Comment:
If we're removing these, I wonder if we need a 'LLVM_FALLTHROUGH' in case others add to it later?